### DISCUSSION TITLE
custom variables how?

### DISCUSSION THREAD

**Discussion by tome9111991** [supermerill/SuperSlicer#1841] [Q&A] [answered]
Hi.
The custom variables feature sounds intressting.
But i dont know how to work/handle.

Maybe someone can write a tutorial? 😅
Some pictures would be nice or video on YT 😀

**Comment by supermerill** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/6536403/141011446-a1010011-3cf3-47cf-b839-66568cdfbf4b.png)
Seems like I made a little mistake: print & printer vars are stored in an array (like for the filament), so they need a [0] to be accessed. Will be fixed.

>  or video on YT

I can fill an entire channel.

**Comment by tome9111991** [comment]
thanks that looks helpful. 

do you plan to update the wiki someday?

**Reply by supermerill** [reply]
I plan to update the slic3r doc.
yes, I think the wiki needs to be freshened up. The homepage, too.

**Reply by tome9111991** [reply]
Is there a way that a group of selected users can help you update the wiki? looks like a lot of work. 😛

**Comment by jimkevern** [comment]
To be clear, apparently custom variables must be defined AND SET in the custom variables section under NOTES.  I tried re-setting it to a different value in the gcode itself with MyCustomVariable = 55 and that didn't appear to work.
